orchestrator/models.py

```diff
@@ -1,28 +1,60 @@
 import uuid
 from enum import IntEnum
 from django.db import models
+from django.utils import timezone
 from django.contrib.postgres.fields import JSONField
 
 
 def message_default():
     message_default_dict = {
         'log': '',
         'failed_jobs': {},
-        'unknown_jobs':{},
+        'unknown_jobs': {},
         'info': ''
     }
     return message_default_dict
 
 
 class Status(IntEnum):
     CREATED = 0
-    PENDING = 1
-    RUNNING = 2
-    COMPLETED = 3
-    FAILED = 4
-    ABORTED = 5
-    UNKNOWN = 6
-    SUSPENDED = 7
+    SUBMITTING = 1
+    SUBMITTED = 2
+    PENDING = 3
+    RUNNING = 4
+    COMPLETED = 5
+    FAILED = 6
+    ABORTED = 7
+    UNKNOWN = 8
+    SUSPENDED = 9
+
```

Comment:
I think this looks great, but just FYI -- I always found https://github.com/viewflow/django-fsm easy to work with